generators/build/templates/Rakefile

```diff
@@ -24,7 +18,22 @@ namespace :static do
   end
 end
 
-task :prereqs => [ 'presteps:clean_up' ]
+namespace :integration do
+  task :ensure do
+    success = system ("dep ensure")
+    if not success 
+      raise "ERROR: Dep ensure failed!\n".red
+    end
+  end
```

Comment:
I feel the restore of pkgs should be in `prep` task.

Reply:
Sounds good.